gulpfile.js

```diff
@@ -32,12 +32,17 @@ gulp.task('test', function (cb) {
     make('testLegacy', cb);
 });
 
+gulp.task('layout', function (cb) {
+    make('build', cb) &&
+    make('layout', cb);
+});
+
 gulp.task('package', function (cb) {
     var publish = process.argv.filter(function (arg) { return arg == '--server' }).length > 0;
     make('build', cb) &&
-        make('package', cb) &&
-        make('test', cb) &&
-        make('testLegacy', cb) &&
-        publish &&
-        make('publish', cb);
+    make('package', cb) &&
```

Comment:
Hey Eric, 
we are now making use of the --version to log the error, instead of throwing it, in-case the version is not present. This fulfills our purpose of getting the aggregated zip layout as well.